src/compass/utils/helpers.py

```diff
@@ -318,3 +318,59 @@ def burst_bboxes_from_db(burst_ids, burst_db_file=None, burst_db_conn=None):
 
     # TODO add warning if not all burst bounding boxes found
     return dict(zip(burst_ids, zip(epsgs, bboxes)))
+
+
+def write_raster(filename, data_list, descriptions,
+                 data_type=gdal.GDT_Float32, format='ENVI'):
+    '''
+    Write a multiband GDAL-friendly raster to disk.
+    Each dataset allocated in the output file contains
+    a description of the dataset allocated for that band
+
+    Parameters
+    ----------
+    filename: str
+        Filepath where to store output dataset
+    data_list: list, np.ndarray
```

Comment:
```suggestion
    data_list: list of np.ndarray
```
Change for clarity - `,` separating the two items gives the wrong impression that it's either or.

Reply:
```suggestion
    data_list: list[np.ndarray]
```
This also works and I think is convention we've been using in COMPASS.